### DISCUSSION TITLE
Input Style Filled is incompatible with dark mode

### DISCUSSION THREAD

**Discussion by CyleClone** [primefaces/community#1581] [PrimeNG Templates]
I'm using the Ultima design and have recently upgraded to PrimeNG 17 and Ultima 17.
I'm offering Light, as well as Dark mode. When switching the modes I'm also switching the input style, but also allowing the user to choose which one they prefer. In light mode it's "Outlined" in dark mode it's "Filled".

1. While the input style is set to "Outlined", checkboxes have a color to make them more noticable. In "Filled" those checkboxes become white with a grey checkmark, which is fine and all. Now we swap to dark mode and that checkbox is fully black.
 ![image](https://github.com/primefaces/community/assets/30074212/fc2eabce-ca50-4e13-a146-e3d76640cc9a)
![image](https://github.com/primefaces/community/assets/30074212/c755d7c2-efdc-4bdd-ac4a-1538e110e769)
now for the first part I did make an override, but overrides shouldn't be necessary. Unfortunately using a checkbox inside a list does not compile my custom styling.

2. when applying a floating label to a dropdown it adds a padding so there's more space. with the input style filled it also applies this padding to children elements. In this case when you have the filter active it also adds padding to that one
Filled: 
![image](https://github.com/primefaces/community/assets/30074212/a8ed395c-9987-4ac1-9fe3-bb19e9699b36)
Outlined:
![image](https://github.com/primefaces/community/assets/30074212/b09c64e6-d7f1-4cec-98ba-9ba2a6c03801)

Somewhat related discussion:
https://github.com/orgs/primefaces/discussions/1006

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for reporting the issue, we'll investigate and get back to you.

**Comment by cetincakiroglu** [comment]
Hi,

We've reviewed your issue, could you please also share a short video recording that shows the bug in action?
